### PR TITLE
updating open hand demo for smoother opening

### PR DESCRIPTION
**Pull request by biofotis**

### CODE REVIEW

File: sr_ethercat_hand_config/demos/demo_lo.py [hunk 1]
```diff
@@ -33,6 +33,14 @@
              'lh_THJ1': 0.0, 'lh_THJ2': 0.0, 'lh_THJ3': 0.0, 'lh_THJ4': 0.0, 'lh_THJ5': 0.0,
              'lh_WRJ1': 0.0, 'lh_WRJ2': 0.0}
 
+unpack_hand_1 = {'lh_THJ1': 0.0, 'lh_THJ2': 0.0, 'lh_THJ3': 0.0, 'lh_THJ4': 1.2, 'lh_THJ5': 0.0}
```
Comment (mikramarc) [inline sr_ethercat_hand_config/demos/demo_lo.py]: If the first state is named open hand, please name this one to sth like open thumb.

File: sr_ethercat_hand_config/demos/demo_lo.py [hunk 2]
```diff
@@ -33,6 +33,14 @@
...
+
+# Open the thumb
```
Comment (mikramarc) [inline sr_ethercat_hand_config/demos/demo_lo.py:38]: if you require comment, its better to wrap the code in a funtion, comments are unreliable.

File: sr_ethercat_hand_config/demos/demo_lo.py [hunk 3]
```diff
@@ -33,6 +33,14 @@
...
+joint_states = unpack_hand_1
```
Comment (mikramarc) [inline sr_ethercat_hand_config/demos/demo_lo.py]: this variable assignment is unnecessary, you can just pass the original var to the method

File: sr_ethercat_hand_config/demos/demo_lo.py (hunk 1 above)
Comment (giusebar) [inline sr_ethercat_hand_config/demos/demo_lo.py]: can we give it a more suitable name? e.g. open_thumb

File: sr_ethercat_hand_config/demos/demo_ro.py [hunk 4]
```diff
@@ -33,6 +33,14 @@
              'rh_THJ1': 0.0, 'rh_THJ2': 0.0, 'rh_THJ3': 0.0, 'rh_THJ4': 0.0, 'rh_THJ5': 0.0,
              'rh_WRJ1': 0.0, 'rh_WRJ2': 0.0}
 
+unpack_hand_1 = {'rh_THJ1': 0.0, 'rh_THJ2': 0.0, 'rh_THJ3': 0.0, 'rh_THJ4': 1.2, 'rh_THJ5': 0.0}
```
Comment (giusebar) [inline sr_ethercat_hand_config/demos/demo_ro.py]: same

**Review by giusebar — CHANGES_REQUESTED** [review]

File: sr_ethercat_hand_config/demos/demo_lo.py [hunk 5]
```diff
@@ -33,6 +33,14 @@
...
+rospy.loginfo("Moving thumb to open position")
+hand_commander.move_to_joint_value_target_unsafe(joint_states, 1.0, False)
+rospy.sleep(1)
+
 # Move hand to open position
 joint_states = open_hand
```
Comment (mikramarc) [inline sr_ethercat_hand_config/demos/demo_lo.py]: same here

File: sr_ethercat_hand_config/demos/demo_lo.py (hunk 3 above)
Comment (giusebar) [inline sr_ethercat_hand_config/demos/demo_lo.py]: also do we need this extra copy?

File: sr_ethercat_hand_config/demos/demo_lo.py (hunk 1 above)
Comment (mikramarc) [reply sr_ethercat_hand_config/demos/demo_lo.py]: or unpack_hand_0 and unpack_hand_1

**Review by giusebar — CHANGES_REQUESTED** [review]

File: sr_ethercat_hand_config/demos/demo_ro.py [hunk 6]
```diff
@@ -17,7 +17,7 @@
 # with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
```
Comment (mikramarc) [inline sr_ethercat_hand_config/demos/demo_ro.py:19]: same comments as in the other file

**Review by mikramarc — CHANGES_REQUESTED** [review]

**Review by mikramarc — APPROVED** [review]

**Review by georgiablanco — APPROVED** [review]

**Review by giusebar — APPROVED** [review]

**Pull request merged**